page-objects/pages/teamsPages/TMSTeamFilesPage.js

```diff
@@ -2,8 +2,6 @@
 'use strict';
 
 const { CLIENT } = require("../../../shared-objects/servers");
+// team internal options
+constfilesTab = 'div[data-testid="team_files"]';
```

Comment:
There's a space missing between const and filesTab.

Reply:
done